server/src/inmantals/server.py

```diff
@@ -53,8 +54,30 @@
 Recent versions use the encapsulating environment and require explicit project installation as a safeguard.
 """
 
-logger = logging.getLogger(__name__)
+BEFORE_ANCHOR_TARGET: bool = CORE_VERSION <= version.Version("8.3.dev")
```

Comment:
I think 8.3 is right but I'm not entirely sure. It is the version of my master branch of core.

Reply:
You could also do the import and catch the exception, that is a very robust way of detecting it.